test/unit/lib/migration-payloads/validation/payload-validation-field-validations.spec.js

```diff
@@ -63,6 +63,32 @@ describe('payload validation', function () {
       ]);
     }));
 
+    it('errors with correct invalid key on unknown validations key', Bluebird.coroutine(function * () {
+      const steps = yield migrationSteps(function up (migration) {
+        const person = migration.createContentType('person')
+          .name('Person')
+          .description('A Person');
+
+        person.createField('fullName')
+          .name('Full Name')
+          .type('Symbol')
+          .validations([{ unique: true }, { size: { min: 5, max: 10 }, somethingUnknown: 'foo' }]);
+      });
+      const chunks = migrationChunks(steps);
+      const plan = migrationPlan(chunks);
+      const payloads = migrationPayloads(plan);
+      const errors = validatePayloads(payloads);
+
+      expect(errors).to.eql([
+        [
+          {
+            type: 'InvalidPayload',
+            message: `A field can't have the combination "size" & "somethingUnknown" as a validation.`
```

Comment:
I find the error message somewhat misleading.  
I guess the problem is that our validations are just simple keys with the actual validation config being the value.  
So the error is more of a 'A validation can only have one key' error or something like that. 

I think ideally if there is one valid and `n` invalid properties we'd say that the valid validation cannot have any other keys in there, if all of them are invalid we just list them all as invalid, and if there are multiple valid ones we mention that they need to be separated.

At the same time it's all more of a nice to have, so I don't insist on it :)

Reply:
> A validation can only have one key

But a validation can have multiple keys, see #23.

Reply:
Oh, right. Damn :(